addons/mail/static/src/core/common/persona_model.js

```diff
@@ -57,8 +57,9 @@ export class Persona extends Record {
         /** @this {import("models").Persona} */
         compute() {
             if (
-                this.type === "guest" ||
-                (this.type === "partner" && this.im_status !== "im_partner" && !this.is_public)
+                this.im_status_access_token &&
```

Comment:
Why do we have to limit to the token being present? It's still possible to gain access through having ACL.

Reply:
I guess the problem if we remove this condition is that we won't re-call onAdd again if the token wasn't available and then becomes available.